wp-includes/theme-compat/error.php

```diff
@@ -16,6 +16,7 @@
 <main>
 	<h1><?php esc_html_e( 'Oops! It looks like you&#8217;re offline.', 'pwa' ); ?></h1>
 	<p><?php esc_html_e( 'Please check your internet connection, and try again.', 'pwa' ); ?></p>
+	<p><!-- WP_OFFLINE_COMMENT --></p>
```

Comment:
@todo This is a placeholder for testing -- the whole offline message part should be replaced with a template tag instead of just adding a new one for offline comment.